src/lib/lesson-comments.ts

```diff
@@ -14,3 +16,61 @@ export async function saveCommentForLesson(slug: string, comment: Comment) {
     )
     .then(({data}) => data)
 }
+
+type GraphQLComment = {
+  comment: string
+  commentable_id: number
+  commentable_type: string
+  created_at: string
+  id: number
+  is_commentable_owner: boolean
+  state: string
+  user: {
+    avatar_url: string
+    full_name: string
+    instructor: {
+      first_name: string
+    }
+  }
+}
+
+export async function loadLessonComments(
+  slug: string,
+  token?: string,
+): Promise<GraphQLComment[]> {
+  token = token || getAccessTokenFromCookie()
+  const graphQLClient = getGraphQLClient(token)
+
+  const variables = {
+    slug: slug,
+  }
+
+  const query = /* GraphQL */ `
+    query getLesson($slug: String!) {
```

Comment:
Load lesson comments separately since those are user-generated content. That will make it easier to swap out the other graphql query once it loads exclusively metadata.

Reply:
suggestion(non-essential): we might consider flipping this and have some sort of comments endpoint instead on the egghead side. Kind of the same thing really, but would be clearer

Reply:
That would be clearer. That will require a small PR to egghead-rails, so I'll do that separately from this PR.